docs/output.md

```diff
@@ -59,15 +74,31 @@ The FastQ reads are mapped to the given reference assembly provided using either
 
 The files resulting from the alignment with graphmap or minimap2 of individual libraries are not saved by default so this directory will not be present in your results. You can override this behaviour with the use of the `--save_align_intermeds` flag in which case it will contain the coordinate sorted alignment files in [`*.bam`](https://samtools.github.io/hts-specs/SAMv1.pdf) format.
 
+![ALIGNER - Alignment per barcode](images/mqc_samtools_alignment_plot_1.png)
+
 *Output directories*:
 
 * `graphmap/`  
-  If the `--aligner graphmap` parameter is used, the results will be output here.
-* `results/minimap2`  
-  If the `--aligner minimap2` parameter is used, the results will be output here.
+  If the `--aligner graphmap` parameter is used, the sorted and indexed bam files will be output here.
+* `minimap2/`  
+  If the `--aligner minimap2` parameter is used, the sorted and indexed bam files will be output here.
 * `<ALIGNER>/samtools_stats/`  
   `*.flagstat`, `*.idxstats` and `*.stats` files generated from the alignment files using SAMtools.
 
+## bigWig and bigBed 
+
+*Documentation*:
+[`BEDTools`](https://github.com/arq5x/bedtools2/), [`bedGraphToBigWig`](http://hgdownload.soe.ucsc.edu/admin/exe/), [`bedToBigBed`](http://hgdownload.soe.ucsc.edu/admin/exe/)
+
+*Description*:
+Creation of bigWig and bigBed coverage tracks for visualisation. This can by bypassed by setting the parameters `--skip_bigwig` and/or `--skip_bigbed`.
```

Comment:
```suggestion
Creation of bigWig and bigBed coverage tracks for visualisation. This can be bypassed by setting the parameters `--skip_bigwig` and/or `--skip_bigbed`.
```